### DISCUSSION TITLE
Github actions stuck on queued

### DISCUSSION THREAD

**Discussion by nuridisCLick** [community/community#30018] [Actions]
Github actions stuck on queued for a long time in private repo, how to fix this bug/issue? 
Yesterday there was a similar bug with GitActions worldwide, but I still facing this issue/bug.

**Comment by radar-blip** [comment]
Someone else is having this same issue, here: #30020
You can file a bug report here: [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by cron410** [comment]
~~Same issue for me at https://github.com/packetworks/onetimesecret/~~

Edit: Triggering a workflow on a different repo in the same account after requesting to cancel the stuck workflow seems to have resolved the issue.
![Screen Shot 2022-08-19 at 1 32 48 PM](https://user-images.githubusercontent.com/3082899/185675426-2a156ebc-5e13-404d-869a-4a0a6d816cf3.png)
